### DISCUSSION TITLE
How to style internal component elements?

### DISCUSSION THREAD

**Discussion by marcodenic** [themesberg/flowbite-react#765] [General]
**Is your feature request related to a problem? Please describe.**
When using a flowbite-react component (drop down menu), I want to style inner elements, specifically the div containing the ul which contains the dropdown items: `<ul class="py-1"> `

I want to have control over these styles.. Why is it adding padding top and bottom which makes the options look weird?

**Describe the solution you'd like**
Id like a way for classNames to pass through to individual inner elements.. how else would you style these components?!

**Comment by rluders** [comment]
https://flowbite-react.com/theme
